myapp/main.py

```diff
@@ -0,0 +1,474 @@
+from flask import Flask, render_template, request, flash, redirect, url_for, jsonify, session, make_response
+from flask_sqlalchemy import SQLAlchemy
+import db, schema, time, ast
+try:
+    import simplejson as json
+except (ImportError):
+    import json
+from utility import Generate_lab_id, Generate_row_id, Generate_data_id
+#Initialize an app
+app = Flask(__name__)
+app.debug = True
+app.secret_key = 'A0Zr98j/3yX R~XHH!jmN]LWX/,?RT'
+
+
+# 1.Commands
+# Utility method for creating all the tables in the database
+@app.route('/admin/bootstrap')
+def bootstrap():
+    schema.Base.metadata.create_all(db.engine)
+    return "Bootsrapped!"
+
+# Clean all the rows in Lab_rows and Lab_desc
+@app.route('/clean_database', methods=['GET'])
+def clean_database():
+    db_session = db.get_session()
+    for r in db_session.query(schema.Lab_rows):
+        db_session.delete(r)
+    for d in db_session.query(schema.Lab_desc):
+        db_session.delete(d)
+    db_session.commit()
+    return "Database has been cleaned"
+
+
+# 2.Common Side
+# common homepage(temporary)
```

Comment:
I'd avoid comments that repeat the code, since they don't provide helpful information, like broader context, or explanation of strange behavior.